hscontrol/types/config.go

```diff
@@ -385,10 +393,18 @@ func validateServerConfig() error {
 		}
 	}
 
+	if viper.IsSet("oidc.use_unverified_email") {
+		log.Warn().
+			Msg("unverified emails will be accepted during oidc authentication (oidc.use_unverified_email=true)")
+	} else {
+		log.Warn().Msg("only verified emails will be accepted during oidc authentication (oidc.use_unverified_email=false)")
+	}
+
```

Comment:
Can you remove this block entirely as it prints a warning for static configuration that does not change during runtime?